seed/php-sdk/imdb/src/Types/ExampleType.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Seed\Types;
+
+use DateTime;
+use Seed\Core\ArrayType;
+use Seed\Core\DateType;
+use Seed\Core\JsonProperty;
+use Seed\Core\SerializableType;
+use Seed\Core\Union;
+
+class ExampleType extends SerializableType
+{
+    /**
+     * @param string[] $stringList
```

Comment:
IDE only complains about arrays not having PHPDocs since those are the only ones that aren't strongly typed directly in the declaration

So, I've only included these here to fix warnings, and skipped adding the non-array types.

However, I have no strong feelings here, and could include all params

Reply:
Sounds good - I actually implemented the comment rendering to include the typed equivalent of `array`. Right now we're representing single dimensional arrays as `array<string>` instead of `string[]`, but we can easily change that [here](https://github.com/fern-api/fern/blob/7f623dd4d5699493e0171c0c019d970eec36e93b/generators/php/codegen/src/ast/Type.ts#L78).

Reply:
The implementation will create all params by default (for consistency and simplicity), but don't feel the need to do it in your hand-written equivalent. The comments will show up once we actually generate things.